samples/samples/pg_snippets.py

```diff
@@ -38,19 +39,19 @@ def create_instance(instance_id):
     spanner_client = spanner.Client()
 
     config_name = "{}/instanceConfigs/regional-us-central1".format(
-      spanner_client.project_name
+        spanner_client.project_name
```

Comment:
Please avoid making changes to other snippets and samples outside of the PR. If their needs to be a lint action to get existing things clean, please do that in it's own PR.

Reply:
Sure @kurtisvg. We'll keep that in mind next time onwards. 

Can you take a look if there's anything else pending, since we are looking to merge these later today.